### DISCUSSION TITLE
Allignment to the left

### DISCUSSION THREAD

**Discussion by herrzettpunkt** [totaldebug/atomic-calendar-revive#734] [Support] [answered]
Hello,

is it possible to move the day/date and the calendar item to the outer left (red lines)?

<img width="302" alt="atomic" src="https://user-images.githubusercontent.com/89742641/176555164-88a8f58d-f95e-40ef-97b6-06727aca6d15.png">

**Comment by marksie1988** [comment, marked as answer]
You would be able to adjust this with card-mod
